tests/integration/targets/vm_list_group_by_clusters/mock.json

```diff
@@ -0,0 +1,106 @@
+[
+{
+  "httpRequest": {
+    "method": "POST",
+    "path": "/rest/com/vmware/cis/session"
+  },
+  "httpResponse": {
+    "statusCode": 200,
+    "body": {"value": "72300ca9ff16c5743fa0a6328c8570ce"}
+  }
+},
+{
+  "httpRequest": {
+    "method": "POST",
+    "path": "/api"
+  },
+  "httpResponse": {
+    "statusCode": 200,
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "body": {"jsonrpc": "2.0", "result": {"output": 0}, "id": "0"}
+  }
+},
+{
+  "httpRequest": {
+    "method": "POST",
+    "path": "/api",
+    "body": {
+      "type": "JSON",
+      "matchType": "PARTIAL",
+      "json": "{\"params\":{\"serviceId\":\"com.vmware.vcenter.cluster\",\"operationId\":\"list\"}}"
+    }
+  },
+  "httpResponseTemplate": {
+    "template": "{\"statusCode\": 200, \"headers\": {\"Content-type\": \"application/json\"}, \"body\": {\"jsonrpc\": \"2.0\", \"result\": {\"output\": [{\"STRUCTURE\": {\"com.vmware.vcenter.cluster\": {\"cluster\": \"cluster1\", \"name\": \"cluster1\", \"drs_enabled\": true, \"ha_enabled\": true}}}]}, 'id': '0'}}",
+    "templateType": "VELOCITY"
+  },
+  "priority": 1
+},
+{
+  "httpRequest": {
+    "method": "POST",
+    "path": "/api",
+    "body": {
+      "type": "JSON",
+      "matchType": "PARTIAL",
+      "json": "{\"params\":{\"serviceId\":\"com.vmware.vcenter.folder\",\"operationId\":\"list\"}}"
+    }
+  },
+  "httpResponseTemplate": {
+    "template": "{\"statusCode\": 200, \"headers\": {\"Content-type\": \"application/json\"}, \"body\": {\"jsonrpc\": \"2.0\", \"result\": {\"output\": [{\"STRUCTURE\": {\"com.vmware.vcenter.folder\": {\"folder\": \"folder1\", \"name\": \"folder1\", \"type\": \"VIRTUAL_MACHINE\"}}}]}, 'id': '1'}}",
+    "templateType": "VELOCITY"
+  },
+  "priority": 1
+},
+{
+  "httpRequest": {
+    "method": "POST",
+    "path": "/api",
+    "body": {
+      "type": "JSON",
+      "matchType": "PARTIAL",
+      "json": "{\"params\":{\"serviceId\":\"com.vmware.vcenter.VM\",\"operationId\":\"list\"}}"
+    }
+  },
+  "httpResponseTemplate": {
+    "template": "{\"statusCode\": 200, \"headers\": {\"Content-type\": \"application/json\"}, \"body\": {\"jsonrpc\": \"2.0\", \"result\": {\"output\": [{\"STRUCTURE\": {\"com.vmware.vcenter.VM\": {\"name\": \"vm1\", \"power_state\": \"POWERED_OFF\", \"vm\": \"vm1\"}}}]}, 'id': '2'}}",
```

Comment:
Please use `${json.parse($!request.body)['id']}'` for `id`, it may work now with hardcoded numbers but, may be flaky in future. same for above calls.

Reply:
It wasn't work for me... that's why I moved to use the ID